src/Mod/Path/PathScripts/PostUtilsParse.py

```diff
@@ -365,12 +379,23 @@ def parse(values, pathobj):
                                 )
                         else:
                             continue
-                    elif param in ["H", "L", "T"]:
+                    elif param in ("H", "L", "T"):
                         outstring.append(param + str(int(c.Parameters[param])))
                     elif param == "D":
-                        if command in ["G41", "G42"]:
+                        if command in ("G41", "G42"):
                             outstring.append(param + str(int(c.Parameters[param])))
-                        elif command in ["G96", "G97"]:
+                        elif command in ("G41.1", "G42.1"):
+                            pos = Units.Quantity(
+                                c.Parameters[param], FreeCAD.Units.Length
```

Comment:
```suggestion
                            pos = Units.Quantity(
                                c.Parameters[param], Units.Length
```
Use one or another.

Reply:
The newly pushed version should have the changes you pointed out in it.